pkg/analyzer/lib/src/dart/element/element.dart

```diff
@@ -2540,6 +2543,12 @@ class ElementAnnotationImpl implements ElementAnnotation {
       element.name == _SEALED_VARIABLE_NAME &&
       element.library?.name == _META_LIB_NAME;
 
+  @override
+  bool get isUnsafe =>
+      element is PropertyAccessorElement &&
```

Comment:
This seems wrong to me. I would have expected that it would be a `ClassElement`.

Reply:
Hint: I don't know what I am doing :-) So thanks for all the help!

I have not setup my environment to test this yet. When the annotation is agreed upon and pushed to master, I can tinker with this.